meta-rdk-mtk-bpir4/recipes-connectivity/dibbler/dibbler_git.bbappend

```diff
@@ -0,0 +1,6 @@
+#To avoid multiple installation
+do_install_append_broadband() {
```

Comment:
broadband suffix is not needed here..this layer supports only broadband.

Reply:
Addressed Keerthana